src/main/java/com/openlattice/data/requests/EntitySetSelection.java

```diff
@@ -38,6 +48,10 @@ public EntitySetSelection(
         this.entityKeyIds = entityKeyIds;
     }
 
+    /**
+     * Creates an EntitySetSelection with with only narrowing down properties.
+     * @param properties The ids of properties to select.
+     */
```

Comment:
This functionality seems to duplicate the above constructor in the case where entityKeyIds is left empty?

Reply:
It's a convenience, for when you only want to provide properties but no entityKeyIds, so you don't have to call the other constructor with Optional.empty()